cmc-domain/src/test/resources/claim-application.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
any reason for this in domain? its for testing the api, so I would have though either an API module or in the core module

Reply:
This is required by `ClaimDataSerializationTest`